### DISCUSSION TITLE
Perhaps we can use criterion as a performance testing tool

### DISCUSSION THREAD

**Discussion by xin2017338** [web-infra-dev/rspack#2957] [Ideas]
![image](https://user-images.githubusercontent.com/28481035/234795905-dce8ff4b-c2bc-482c-b122-d2dc1139e02b.png)
Perhaps we can use criterion as a performance testing tool

**Comment by hyf0** [comment]
Yeah. You are right and we have one as said in https://github.com/web-infra-dev/rspack/pull/2956.
